### PR TITLE
Add compare elements

### LINKED ISSUE #307: How to compare elements with each other
**Rod Version:** v0.84.1

## The code to demonstrate your question

```go
func getIndexOfSliceValue(slice []*rod.Element, value *rod.Element) int {
	var index int
	for i := 0; i < len(slice); i++ {
		if value.MustHTML() == slice[i].MustHTML() {
			index = i
			break
		}

	}
	return index
}
```

## What have you tried to solve the question

Currently, I have been using the code above to get the index number of an element from an element list, it works, but I think it must be time and resource consuming to use MustHTML() to compare elements.

### PR DESCRIPTION
**Pull request by infinitepr0**
Fix #307

### CODE REVIEW

File: element.go [hunk 1]
```diff
@@ -664,6 +664,11 @@ func (el *Element) Evaluate(opts *EvalOptions) (*proto.RuntimeRemoteObject, erro
 	return el.page.Context(el.ctx).Evaluate(opts.This(el.Object))
 }
 
+// Equal checks if the two elements are equal.
+func (el *Element) Equal(elm *Element) bool {
+	return el.MustEval(`elm => this === elm`, elm.Object).Bool()
```
Comment (ysmood) [inline element.go:669]: Please don't use Must inside non-Must methods. Check how other methods handle errors, such as `Element` and `MustElement`.

https://go-rod.github.io/#/error-handling

**Review by ysmood — CHANGES_REQUESTED** [review]

**Comment by infinitepr0** [comment]
@ysmood Is `MustEqual` required? I can't think of any case where `Equal()` returns an error.

**Comment by ysmood** [comment]
Yes, when people don't want to use panic, especially for senior developers, it's critical.

**Comment by infinitepr0** [comment]
Ah, okay. Fine then!

**Comment by infinitepr0** [comment]
Can you please rerun tests @ysmood ? They seemed to have timed out.

**Comment by ysmood** [comment]
```
git commit --allow-empty -m "Trigger Build"
```

Push an empty commit to trigger CI.

**Review by ysmood — APPROVED** [review]

**Pull request merged**